packages/api/src/builder/routes.ts

```diff
@@ -1,8 +1,8 @@
 import {fromHexString, toHexString} from "@chainsafe/ssz";
-import {ssz, allForks, bellatrix, Slot, Root, BLSPubkey} from "@lodestar/types";
+import {ssz, allForks, bellatrix, Slot, Root, BLSPubkey, ValidatorIndex, WithdrawalIndex} from "@lodestar/types";
```

Comment:
This file is dedicated to [`builder api`](https://ethereum.github.io/builder-specs/#/). The endpoint you are implementing is for `beacon api`. Although they both share the same `/eth/v1/builder/` route, for the purpose for spec compliant and ease of testing, please create a new file `packages/api/src/beacon/routes/builder.ts` and move all the `getExpectedWithdrawals` definition to there